package.json

```diff
@@ -62,63 +62,64 @@
   "dependencies": {
     "frontend-collective-react-dnd-scrollzone": "^1.0.1",
```

Comment:
Update once https://github.com/frontend-collective/frontend-collective-react-dnd-scrollzone/pull/3 is merged